public/content/contributing/translation-program/faq/index.md

```diff
@@ -56,7 +56,7 @@ If you want to raise a flag on a particular string that needs attention, feel fr
 
 ## What is Translation Memory (TM)? {#translation-memory}
 
-Translation Memory (TM) is a feature of Crowdin that stores all the previously translated strings across [ethereum.org](http://ethereum.org/). When a string is translated, it is automatically saved into our project TM. This could be a useful tool to help you save your time!
+Translation Memory (TM) is a feature of Crowdin that stores all the previously translated strings across [ethereum.org](/). When a string is translated, it is automatically saved into our project TM. This could be a useful tool to help you save your time!
```

Comment:
Tiny non-blocking note, may be worth separately considering the value of the `/` links when we already (and always) have the home link in the nav bar logo... could potentially just remove these. (out of scope here though)